packages/react-charting/src/components/StackedBarChart/MultiStackedBarChart.base.tsx

```diff
@@ -306,7 +310,20 @@ export class MultiStackedBarChartBase extends React.Component<IMultiStackedBarCh
       );
     });
     if (this.props.variant === MultiStackedBarChartVariant.AbsoluteScale) {
-      if (!this.props.hideLabels) {
+      let showLabel = false;
```

Comment:
This block of computation can go inside the !this.prop.hideLabels block.

Reply:
done